src/Components/Profile/ChangePassword.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import "./UserProfile.css"
+import { ChangeUserPassword } from '../../Auth/Actions/userActions'
+import { connect } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+import { StepOne, StepTwo } from './Steps'
+
+
+
+const ChangePassword = ({ChangeUserPassword})=> {
+    const history = useHistory()
+    const [currentStep, setCurrentStep] = useState(0)
+    const [data, setData] = useState({
+        previousPassword: "",
+        newPassword: "",
+        confirmPassword: ""
+    });
+    const makeRequest = (formData) => {
+        ChangeUserPassword(formData, history)
+    }
+
+    const handleNextStep = (newData, final = false) =>{
+        setData(prev => ({...prev, ...newData}))
+        if (final) {
+            makeRequest(newData)
```

Comment:
await the makeRequest and also handle error, use this package `toastify` to notify errors